### PR TITLE
PARQE-5

### PR DESCRIPTION
**Pull request by aschaaf1**
[PARQE-5](https://monarchcapstoneparqe.atlassian.net/browse/PARQE-5)
- created basic user webpage that allows users to upload a file in .stl, .stp, .step, or .3mf format
- file does not currently go anywhere when submit button is clicked
         - Browser is not allowed to directly access file system of users computer
         - Requires backend code to accomplish
         - I marked this task as blocked for now since backend is outside intended scope of this story
         

Screenshot 2023-11-02 at 1 36 49 PM

[PARQE-5]: https://monarchcapstoneparqe.atlassian.net/browse/PARQE-5?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by RyanKneeya — APPROVED** [review]
Looks like a solid start to the prototype to me. Has the file types, input box, and standard header

File: userAPI/style.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+body {
```
Comment (michdopdx) [inline userAPI/style.css:1]: Thanks for adding the css, the page looks much nicer!

**Review by michdopdx — APPROVED** [review]
Looks good man!

**Comment by michdopdx** [comment]
If you can squash and merge that would be great. I think squashing the commits would help with git history organization.

**Comment by aschaaf1** [comment]
> Looks good man!

Thanks! It's pretty basic but it should be something to get us started at least!

**Pull request merged**